code/datums/greyscale/json_reader.dm

```diff
@@ -0,0 +1,52 @@
+/// Takes a json list and extracts a single value.
+/// Subtypes represent different conversions of that value.
+/datum/json_reader
+
+/datum/json_reader/proc/ReadJson(value)
```

Comment:
Document as subtypes use it, also please add in a `return`, it doesn't do anything but it just looks more correct to me 😛